prov/gni/include/gnix_util.h

```diff
@@ -99,6 +99,8 @@ extern atomic_t gnix_debug_next_tid;
 #define GNIX_DEBUG(subsystem, ...)                                             \
 	GNIX_LOG_INTERNAL(FI_DBG, subsystem, __VA_ARGS__)
 #define GNIX_ERR(subsystem, ...)                                               \
+	GNIX_LOG_INTERNAL(GNIX_FI_PRINT, subsystem, __VA_ARGS__)
```

Comment:
I think GNIX_ERR should tack on some extra identifier or something in the string it prints out, so we can distinguish it from GNIX_WARN.  I think the best way to do that is to add something to GNIX_LOG_INTERNAL.